utils/templates/test_module+member.yml.in

```diff
@@ -0,0 +1,88 @@
+---
+- name: Test $name
+  hosts: ipaserver
+  become: true
+
+  tasks:
+
+  # CLEANUP TEST ITEMS
+
+  - name: Ensure $name NAME is absent
+    ipa$name:
+      name: NAME
+      state: absent
+
+  # CREATE TEST ITEMS
+
+  # TESTS
+
+  - name: Ensure $name NAME is present
+    ipa$name:
+      name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.